bugsnag-plugin-android-ndk/src/main/jni/utils/migrate.h

```diff
@@ -1,12 +1,38 @@
 #ifndef BUGSNAG_MIGRATE_H
 #define BUGSNAG_MIGRATE_H
 
-#include "report.h"
+#include "event.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+typedef struct {
+    char name[64];
+    char version[16];
+    char url[64];
+} bsg_library;
+
+/** a Bugsnag exception */
+typedef struct {
+    /** The exception name or stringified code */
+    char name[64];
+    /** A description of what went wrong */
+    char message[256];
+    /** The variety of exception which needs to be processed by the pipeline */
+    char type[32];
+
+    /**
+     * The number of frames used in the stacktrace. Must be less than
+     * BUGSNAG_FRAMES_MAX.
+     */
+    ssize_t frame_count;
+    /**
+     * An ordered list of stack frames from the oldest to the most recent
+     */
+    bsg_stackframe stacktrace[BUGSNAG_FRAMES_MAX];
+} bsg_exception;
+
```

Comment:
Should we include some comments in this file (or rename the type) to make it clear that `bsg_exception` and `bsg_library` are for older version?

Reply:
Added some general documentation towards the top which should be helpful for future readers.